src/page/NewPassword/NewPassword.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useRef, useEffect } from "react";
+import './NewPassword.scss';
+import { NamePage } from "../../components/NamePage/NamePage";
+import { SignForm } from '../../components/SignForm/SignForm';
+import { Input } from "../../components/Input";
+import { Button } from "../../components/Button";
+
+export const NewPassword = () => {
+    const [valuePassword, setValuePassword] = useState('');
+    const [errorPassword, setErrorPassword] = useState('');
+    const [valueConfirmPassword, setValueConfirmPassword] = useState('');
+    const [errorConfirmPassword, setErrorConfirmPassword] = useState('');
+
+    const inputPassword = React.useRef<HTMLInputElement>(null);
+    const inputConfirmPassword = React.useRef<HTMLInputElement>(null);
+
+    const changeInputPassword = (event: any): void => setValuePassword(event.target.value)
+    const changeInputConfirmPassword = (event: any): void => setValueConfirmPassword(event.target.value)
+
+    const handleSubmit = () => {
+
+    }
+
+    useEffect(() => {
+       inputPassword.current?.addEventListener('focus', () => setErrorPassword(''));
+
+        {valueConfirmPassword != valuePassword
+        ?
+        setErrorConfirmPassword("Passwords don't match")
+        :
+        setErrorConfirmPassword('')}
+
+        return () => {
+            inputPassword.current?.removeEventListener('focus', () => setErrorPassword(''));
+        }
+    })
```

Comment:
Такая запись не сработает для сброса события + навешивать и снимать события нужно в хуке с []
```suggestion
    useEffect(() => {
       const focusHandler = () => setErrorPassword('')
       inputPassword.current?.addEventListener('focus', focusHandler);

        {valueConfirmPassword != valuePassword
        ?
        setErrorConfirmPassword("Passwords don't match")
        :
        setErrorConfirmPassword('')}

        return () => {
            inputPassword.current?.removeEventListener('focus', focusHandler);
        }
    }, [])
```